src/workerd/api/tests/js-rpc-test.js

```diff
@@ -1,4 +1,5 @@
 import assert from 'node:assert';
+import { waitUntil } from 'cloudflare:workers';
```

Comment:
The test should likely also test the error condition of calling `waitUntil` at the global scope / outside of an IoContext.

Reply:
ha! I refreshed then saw that you had that :-) .. nevermind!